e2e-tests/fixtures/Plan.ts

```diff
@@ -48,7 +48,7 @@ export class Plan {
    */
   async goto() {
     await this.page.goto('/plans');
-    await this.page.waitForTimeout(1000);
+    await this.page.waitForTimeout(1200);
```

Comment:
is this periodically failing so you had to bump up the timeout to make it pass more consistently?

Reply:
Yes. The reason is a little involved. Basically when we create a model we have Hasura events that fire to fill the activity types. Those Hasura events take a small amount of time and there is no way to tell when they complete. So we just have to do this busy waiting before loading the plan. If we try to load a plan before the model has populated the activity types bad things happen.